product/tfe.go

```diff
@@ -51,5 +51,12 @@ func tfeRunners(cfg Config, api *client.APIClient) ([]runner.Runner, error) {
 		runner.NewHTTPer(api, "/api/v2/admin/twilio-settings", nil),
 		// page size 1 because we only actually care about total workspace count in the `meta` field
 		runner.NewHTTPer(api, "/api/v2/admin/workspaces?page[size]=1", nil),
+
+		runner.NewCommander("docker -v", "string", nil),
+		runner.NewCommander("replicatedctl app status --output json", "json", nil),
+		runner.NewCommander("lsblk --json", "json", nil),
+
+		runner.NewSheller("getenforce", nil),
+		runner.NewSheller("echo $proxy_ip", nil),
```

Comment:
Do we expect `proxy_ip` to already be set/exported by something else? If so, what is that? I would expect us to look for `HTTP_PROXY` or `NO_PROXY` env vars instead.

Reply:
Oh whoops, you're right. That was an oversight. Should I look for both `HTTP_PROXY` and `HTTPS_PROXY`?

Reply:
We talked about this out of band. Landed on `env | grep -i proxy`. Marking this resolved.